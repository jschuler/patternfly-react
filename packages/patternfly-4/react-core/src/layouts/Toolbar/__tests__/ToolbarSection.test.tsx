// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { ToolbarSection } from '../ToolbarSection';

describe('ToolbarSection test', () => {
  it('ToolbarSection should match snapshot', () => {
    const view = shallow(
      <ToolbarSection
        children={<>ReactNode</>}
				className={"null"}
				aria-label={undefined}
      />);
    expect(view).toMatchSnapshot();
  });
});
