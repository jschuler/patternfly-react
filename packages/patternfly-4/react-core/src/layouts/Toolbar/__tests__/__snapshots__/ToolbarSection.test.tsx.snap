// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolbarSection test ToolbarSection should match snapshot 1`] = `
<section
  className="pf-l-toolbar__section null"
>
  ReactNode
</section>
`;
